ContactDetailsApi/appsettings.json

```diff
@@ -1,9 +1,9 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Information",
-      "System": "Information",
-      "Microsoft": "Information"
+      "Default": "Trace",
+      "System": "None",
+      "Microsoft": "None"
```

Comment:
I would leave this as it is and not change it for the time being.
We probably need to have a conversation with Mirela soon about how we dynamically change app settings when deploying to an environment without having to make code changes.